client/src/styles/styles.jsx

```diff
@@ -0,0 +1,156 @@
+import styled from 'styled-components';
+
+const DivCalendar = styled.div`
+  height:100%;
+  width:100%;
+  display:flex;
+  flex-flow:column;
+  align-items: center;
+`;
+
+const DivNav = styled.div`
+  display:flex;
+  flex-flow:row;
+  justify-content: center;
+  align-items: center;
+  height:15%;
+  width:90%;
+`;
+
+const DivBanner = styled.div`
+  width:72%;
+  height:100%;
+  display:flex;
+  flex-flow:row;
+  align-items: center;
+  font-size: 22px;
+  justify-content: center;
+`;
+
+const DivArrowAfter = styled.div`
+  &:hover {
+    opacity: 1;
+  }
+  border:1px solid grey;
+  width:14%;
+  height:90%;
+  background-image:url('http://localhost:3001/rooms/00/arrow.png');
+  background-size: 75% 75%;
+  background-repeat: no-repeat;
+  border-radius: 3px;
+  background-position: 50% 50%;
+  opacity: .6;
+  margin-bottom: 5px;
+  transform: rotate(180deg);
+`;
+
+const DivArrowPrev = styled.div`
+  &:hover {
+    opacity: 1;
+  }
+  border:1px solid grey;
+  width:14%;
+  height:90%;
+  background-image:url('http://localhost:3001/rooms/00/arrow.png');
+  background-size: 75% 75%;
+  background-repeat: no-repeat;
+  border-radius: 3px;
+  background-position: 50% 50%;
+  opacity: .6;
+  margin-bottom: 5px;
+`;
+
+const RegularDay = styled.span`
```

Comment:
Make sure that you catch all of the styled components that you can reuse. This is a little bit of refactoring, but it makes a difference in the cleanliness of your code.